src/commands/profiles/delete.ts

```diff
@@ -184,14 +184,13 @@ export default class ProfileDelete extends Command {
     const database = client.database("agid-documentdb-test");
 
     // apply processDeleteOpt to each delete operation
-    const results = await sequential(
-      deleteOpsToProcess,
-      item => this.processDeleteOpt(database, storageConnection, item),
-      []
+    const results = await sequential(deleteOpsToProcess, item =>
+      this.processDeleteOpt(database, storageConnection, item)
+    );
+    const countDeleteItems = results.reduce(
+      (acc, current) => acc + current.fold(0, x => x),
```

Comment:
that fold can be replaced by `getOrElse(0)`, also you can now use `sequentialSum` here